server/tests/backends/fixtures/ifthenelse/quotes.json

```diff
@@ -17,7 +17,7 @@
           "operator": "gt"
         },
         "newColumn": "COND",
-        "then": "'tintin'",
+        "then": "\"tin'tin\"",
         "else": "some ' stuff with \" annoying 'quotes'"
```

Comment:
Shouldn't this else example be invalid? (there is column `some` nor `stuff`, etc...)

Reply:
100% aligné. IMO il faut qu'on rende bcp plus robuste notre formula step. Et plus tôt on le fait sur la v3 mieux on se portera